docs/book/advanced-guide/practical/practical-mlops.md

```diff
@@ -2,6 +2,11 @@
 description: Practical MLOps with ZenML
 ---
 
-#  Things to change
+This advanced guide contains some practical guides and motivating examples
+showcasing the core concepts and features of MLOps. Here you will learn how
+about some of the essential stack components that you will likely use as part of
+your stack, from experiment trackers to secrets manager.
 
```

Comment:
Maybe add a list of lessons here like ZenBytes with the corresponding ZenBytes link?